src/logic/widelands.h

```diff
@@ -62,6 +65,43 @@ constexpr DescriptionIndex INVALID_INDEX = std::numeric_limits<uint16_t>::max();
 constexpr DescriptionIndex kInvalidWare = INVALID_INDEX - 1;
 constexpr DescriptionIndex kNoResource = INVALID_INDEX - 1;
 
+class WarePriority {
```

Comment:
Can this guy live where the requests live? This file is includes in the editor code and in the players manager, for example, where we don't need ware priority.